deploy/helm/trickster/templates/configmap.yaml

```diff
@@ -119,9 +128,14 @@ data:
             {{- if eq .type "redis" }}
             {{ printf "[caches.%s.redis]" .name }}
             protocol = {{ .redis.protocol | quote }}
-            endpoint = {{ .redis.endpoint | quote }}
+            endpoints = {{ .redis.endpoint | quote }}
             password = {{ .redis.password | quote }}
 
+            {{- if eq .type "redis_cluster" }}
+            {{ printf "[caches.%s.redis_cluster]" .name }}
+            endpoint = {{ .redis_cluster.endpoints }}
```

Comment:
you're using `endpoints =` on line 131 but `endpoint =` on 136. Just want to make sure this is intentional, as on the surface it appears to be breaking.